argocd/metadata.csv

```diff
@@ -0,0 +1,152 @@
+argocd.app_controller.go.gc.duration.seconds.count,count,,second,,The summary count of garbage collection cycles in the Repo Server Application Controller,0,argocd,app_controller go gc duration seconds count,
+argocd.app_controller.go.gc.duration.seconds.sum,gauge,,second,,The sum of the pause duration of garbage collection cycles in the Application Controller,0,argocd,app_controller go gc duration seconds sum,
+argocd.app_controller.go.goroutines,gauge,,,,The number of goroutines that currently exist in the Application Controller,0,argocd,app_controller go goroutines,
+argocd.app_controller.go.memstats.alloc_bytes,gauge,,byte,,The number of bytes allocated and still in use in the Application Controller,0,argocd,app_controller go memstats alloc_bytes,
+argocd.app_controller.go.memstats.buck_hash.sys_bytes,gauge,,byte,,The number of bytes used by the profiling bucket hash table in the Application Controller,0,argocd,app_controller go memstats buck_hash sys_bytes,
+argocd.app_controller.go.memstats.frees,count,,,,The notal number of frees in the Application Controller,0,argocd,app_controller go memstats frees,
+argocd.app_controller.go.memstats.gc.cpu_fraction,gauge,,fraction,,The fraction of this program's available CPU time used by the GC since the program started in the Application Controller,0,argocd,app_controller go memstats gc cpu_fraction,
+argocd.app_controller.go.memstats.gc.sys_bytes,gauge,,byte,,The number of bytes used for garbage collection system metadata in the Application Controller,0,argocd,app_controller go memstats gc sys_bytes,
+argocd.app_controller.go.memstats.heap.alloc_bytes,gauge,,byte,,The number of heap bytes allocated and still in use in the Application Controller,0,argocd,app_controller go memstats heap alloc_bytes,
+argocd.app_controller.go.memstats.heap.idle_bytes,gauge,,byte,,The number of heap bytes waiting to be used in the Application Controller,0,argocd,app_controller go memstats heap idle_bytes,
+argocd.app_controller.go.memstats.heap.inuse_bytes,gauge,,byte,,The number of heap bytes that are in use in the Application Controller,0,argocd,app_controller go memstats heap inuse_bytes,
+argocd.app_controller.go.memstats.heap.objects,gauge,,object,,The number of allocated objects in the Application Controller,0,argocd,app_controller go memstats heap objects,
+argocd.app_controller.go.memstats.heap.released_bytes,gauge,,byte,,The number of heap bytes released to OS in the Application Controller,0,argocd,app_controller go memstats heap released_bytes,
+argocd.app_controller.go.memstats.heap.sys_bytes,gauge,,byte,,The number of heap bytes obtained from system in the Application Controller,0,argocd,app_controller go memstats heap sys_bytes,
+argocd.app_controller.go.memstats.last_gc_time.seconds,gauge,,second,,The number of seconds since 1970 of last garbage collection in the Application Controller,0,argocd,app_controller go memstats last_gc_time second,
```

Comment:
```suggestion
argocd.app_controller.go.memstats.last_gc_time.seconds,gauge,,second,,The number of seconds since Unix epoch of last garbage collection in the Application Controller,0,argocd,app_controller go memstats last_gc_time second,
```